pandas/core/arrays/interval.py

```diff
@@ -436,7 +436,7 @@ def from_arrays(cls, left, right, closed="right", copy=False, dtype=None):
             ),
         )
     )
-    def from_tuples(cls, data, closed="right", copy=False, dtype=None):
+    def from_tuples(cls, data, closed="right", dtype=None):
```

Comment:
why remove this?

Reply:
I thought it was unused, but I guess it is used by `from_arrays` in the return.

Reply:
It's actually unused, in the return statement it's hardcoded.

Reply:
this is still an api change, would break things for users who do pass it

Reply:
But then the solution would not to keep it right? I think it's not desirable to have an argument in your function which doesn't have a functionality.

Reply:
If this is public API, then I believe we should provide deprecation warning first?
```
def from_tuples(cls, data, closed="right", copy=None, dtype=None):
    if copy is not None:
        msg = "Setting copy is deprecated and will be removed in version xxx. Setting copy=False"
        warnings.warn(msg, DeprecationWarning)
```
Who would decide that the deprecation of the dead kwarg should be started?

Reply:
revert this change